src/components/MainWalletView.tsx

```diff
@@ -61,7 +69,13 @@ const WalletHeaderRescanning = ({ walletName, isLoading }: { walletName: string;
           <rb.Placeholder className={styles.subtitlePlaceholder} />
         </rb.Placeholder>
       ) : (
-        <h2>{t('current_wallet.text_rescan_in_progress')}</h2>
+        <h2>
+          {serviceInfo?.rescanProgress !== undefined
+            ? t('current_wallet.text_rescan_in_progress_with_progress', {
+                progress: Math.round(serviceInfo.rescanProgress * 100),
```

Comment:
```suggestion
                progress: Math.floor(serviceInfo.rescanProgress * 100),
```

Should we `floor` here?

Reply:
I didn't consider backwards compatibility. Will make the changes and resubmit